docs/index.html

```diff
@@ -1402,7 +1402,7 @@ <h3 id="oauth1">OAuth1</h3>
 <li>Zapier makes a call to your API requesting a &quot;request token&quot; (also known as &quot;temporary credentials&quot;).</li>
 <li>Zapier sends the user to the authorization URL, defined by your app, along with the request token.</li>
 <li>Once authorized, your website sends the user to the <code>redirect_uri</code> Zapier provided. Use <code>zapier describe</code> command to find out what it is: <img src="https://zappy.zapier.com/117ECB35-5CCA-4C98-B74A-35F1AD9A3337.png" alt></li>
-<li>Zapier makes a backend call to your API to your API to exchange the request token for an &quot;access token&quot; (also known as &quot;long-lived credentials&quot;).</li>
+<li>Zapier makes a backend call to your API to exchange the request token for an &quot;access token&quot; (also known as &quot;long-lived credentials&quot;).</li>
```

Comment:
Not sure why these are committed, these are a subset of changes from https://github.com/zapier/zapier-platform/commit/6a7c0a0fbf603a6c23c3d29da545bafbdf907feb 
Could be related to pre-commit hooks?